visualization/app/codeCharta/ui/copyToClipboardButton/util/clipboardString.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { buildTextOfFiles } from "./clipboardString"
+import { FileToValue } from "./getFilenamesWithHighestMetrics"
+
+describe("buildTextOfFiles", () => {
+	it("should return only header line if there is only one attribute without files", () => {
+		const result = buildTextOfFiles(WITH_ONE_FILELESS_ATTRIBUTE)
+
+		expect(result).toBe("FUNCTIONS\n")
+	})
+	it("should return valid string if there two attributes", () => {
+		const result = buildTextOfFiles(WITH_TWO_ATTRIBUTES)
+
+		expect(result).toBe(
+			`RLOC\n` + `\t${String.fromCodePoint(8226)} fileA (12)\n` + `COMMENTS\n` + `\t${String.fromCodePoint(8226)} fileA (14)\n`
```

Comment:
as far as I know node supports utf-8. Maybe also utf-16? Can we just copy paste the real char here?

Reply:
Resolved with commit [89ab875](https://github.com/MaibornWolff/codecharta/pull/2942/commits/89ab875127f1d597acf125a17d0178e19e5096b9)